lib/prepare_release.js

```diff
@@ -204,6 +210,32 @@ class ReleasePreparation {
     return this.config.username;
   }
 
+  warnForMissing() {
+    const { cli, upstream, username } = this;
+
+    const missing = !username || !upstream;
+    if (!upstream) {
+      cli.warn('You have not told git-node the remote you want to sync with.');
+      cli.separator();
+      cli.info(
+        'For example, if your remote pointing to nodejs/node is' +
+        ' `remote-upstream`, you can run:\n\n' +
+        '  $ ncu-config set upstream remote-upstream');
+      cli.separator();
+      cli.setExitCode(1);
+    } else if (!username) {
```

Comment:
If both username and upstream are missing should both warnings be output?

Reply:
i sought to emulate `warnForMissing` in session but but i agree with you - better to output both 🤔